packages/xo-web/src/common/select-objects.js

```diff
@@ -126,15 +129,43 @@ class GenericSelect extends React.Component {
     ]).isRequired,
   }
 
-  _getObjectsById = createSelector(
+  _getObjectsById = createSelector(() => this._getObjects(), getObjectsById)
```

Comment:
I don't know why it breaks when i just pass the function instead of creating a new one.

Reply:
Because `_getObjects` is declared after.